.githooks/install-hooks.sh

```
@@ -0,0 +1,35 @@
+
+if [ -z "$REPO_ROOT" ]; then
+    echo "❌ Error: Not in a git repository"
+    exit 1
+fi
+
+cd "$REPO_ROOT" || exit 1
+
+# Set the hooks path
+git config core.hooksPath .githooks
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)